src/styles.py

```diff
@@ -65,9 +65,20 @@ def __init__(self, application: Graphs.Application):
                 _compare_styles,
             )
         enumerator.close(None)
-        # TODO: add System style preview
-        self._system_style = Graphs.Style.new(_("System"), None, None, False)
         self._update_system_style()
+        system_style = self._system_style_name
```

Comment:
Given this is accessed twice assigning it locally is a bit redundant here. I'd just use `self._system_style_name`